_updatePublisher.sh

```diff
@@ -117,13 +117,16 @@ if [[ $skipPrompts == true ]] || [[ $response =~ ^[yY].*$ ]]; then
 
   curl -L $gencont_sh_url -o /tmp/_gencontinuous.new
   cp /tmp/_gencontinuous.new _gencontinuous.sh
+  chmod +x _gencontinuous.sh
```

Comment:
You have make this as an executable here. But when we upgrade IG Publisher, we need to update these scripts which align to the new version. then we can't guarantee those changes will be in the new version as well.
So, I think we need not to make those as executables.
Same for all changes in the scripts.

Reply:
The script changes are generated by the scripts themselves. I.e. that’s a new version of the script from HL7. 

I can double check, but am fairly sure I haven’t done anything manual here.

Reply:
Okey, 
BTW, Where are we specifying which version of the IG publishers is used for this IG?

Reply:
We don't specifically specify version of the IG-publisher. The versioning and CM of the IG publisher in general isn't all that well defined, but as a rule of thumb we use the latest version, since it's the latest version that runs on the HL7 auto-build server. In this case I had to update my local version of publisher in order to reproduce the blocking error we were getting from the auto-build server.

And the "only" way to know which version was used during a build is in the QA section of a built IG: https://commonprofiles.care/fhir/1.0.1/qa.html
Maybe it can be added as a variable to the IG itself if it's relevant?